flink/flink.sh

```diff
@@ -92,7 +92,8 @@ function install_flink_snapshot() {
   tar -xzvf "${flink_local}"
   rm "${flink_local}"
   # only the first match of the flink toplevel pattern is used
-  local files=( ${flink_toplevel_pattern} )
+  local files
+  mapfile -t files < <(compgen -G "${flink_toplevel_pattern}")
```

Comment:
Break it down into 2 steps:

compgen -G "${flink_toplevel_pattern}" >/tmpfile
mapfile -t files </tmpfile

Reply:
Done.